lib/checks/isQuadratic.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const Node = require('../node');
+
+// TODO(ael)
+function isQuadratic(node) {
+  if (!Node.Type.isOperator(node) || node.op !== '+') {
```

Comment:
you can now do `if (!Node.Type.isOperator(node, '+')` thanks to a recent change :)